clang/lib/CodeGen/CGCall.cpp

```diff
@@ -5612,6 +5612,10 @@ RValue CodeGenFunction::EmitCall(const CGFunctionInfo &CallInfo,
                               BundleList);
     EmitBlock(Cont);
   }
+  if (CI->getCalledFunction() && CI->getCalledFunction()->hasName() &&
+      CI->getCalledFunction()->getName().contains("Z4sqrt")) {
```

Comment:
Probably should check the language, and exact match on the permissible names

Reply:
the language is checked by `SetSqrtFPAccuracy`. I'll have a look at a better match for the name